napalm_logs/config/opengear/RXDATA.yml

```diff
@@ -0,0 +1,13 @@
+#<30>ztp-tankstelle_1-ts port02 2018-Nov- 9 15:38:25.341 RXDATA: tankstelle_1-port2-dev#
+
+messages:
+  - error: RXDATA
```

Comment:
What information does `RXDATA` provide?

Reply:
RXDATA contains the raw line of data which the device attached to the console server serial port sent